tests/Makefile.am

```diff
@@ -9,12 +9,13 @@ else
 DBGFLAGS = -g -DNDEBUG
 endif
 
-CFLAGS_GTEST =
-LDADD_GTEST =
+CFLAGS_GTEST = -I $(top_srcdir)/../sonic-swss-common/googletest/googletest/include
```

Comment:
>$(top_srcdir)/.. [](start = 18, length = 16)

We should not refer any files outside the $(top_srcdir)

Reply:
We  only have google unit test environment in swss-common.   What is your suggestion on how to build and run unit test in swss?

Reply:
https://github.com/qiluo-msft/SONiC/wiki/Introduction-to-unit-tests-process-in-SONiC-repos#prepare-google-test-environment-2-options

Reply:
Ok, the question is do we want to build unit test binary automatically like that in swss-common?  Or it is ok just put unit test code there, whoever is interested in running  the unit test can figure out how to build  and run it themselves? 

The other option is to make gtest common library like swss-common.

Reply:
agree that we shall not include files outside this repository. I think the option 1 in the link Qi provided is preferred.

Reply:
We plan to remove googletest from sonic-swss-common repo, and use installed Google Test DEB package near future.
It's ok to build the test if easy.